<ui version="4.0" >
 <class>AirportsWidget</class>
 <widget class="QWidget" name="AirportsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTableView" name="airportsTable" >
         <property name="windowModality" >
          <enum>Qt::NonModal</enum>
         </property>
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLineEdit" name="filter_lineEdit" />
         </item>
         <item>
          <widget class="QPushButton" name="filter_pushButton" >
           <property name="text" >
            <string>filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="icao_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text" >
            <string>ICAO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="icao_lineEdit" >
           <property name="maxLength" >
            <number>32767</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="iata_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>IATA</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="iata_lineEdit" >
           <property name="maxLength" >
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="name_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="name_lineEdit" >
           <property name="maxLength" >
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="city_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>City</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="city_lineEdit" />
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="country_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" >
          <widget class="QLabel" name="latitude_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Latitude</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QSpinBox" name="latitude_spinBox_deg" >
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="latitude_spinBox_min" >
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="latitude_spinBox_sec" >
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="latitude_lineEdit_dir" >
             <property name="maximumSize" >
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>16</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="6" column="0" >
          <widget class="QLabel" name="lonitude_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Longitude</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QSpinBox" name="longitude_spinBox_deg" >
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="longitude_spinBox_min" >
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="longitude_spinBox_sec" >
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="longitude_lineEdit_dir" >
             <property name="maximumSize" >
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>16</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="7" column="0" >
          <widget class="QLabel" name="altitude_label" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Altitude</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QSpinBox" name="altitude_spinBox" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="suffix" >
              <string/>
             </property>
             <property name="maximum" >
              <number>99999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="4" column="1" >
          <widget class="QComboBox" name="country_comboBox" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>298</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pushButton_new" >
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../qAirLog.qrc" >
             <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_delete" >
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../qAirLog.qrc" >
             <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_revert" >
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../qAirLog.qrc" >
             <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_submit" >
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../qAirLog.qrc" >
             <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qAirLog.qrc" />
 </resources>
 <connections/>
</ui>
